docs/preview/02-Guidance/Service-to-service Correlation/use-service-to-service-correlation-in-web-api.md

```diff
@@ -0,0 +1,312 @@
+![HTTP correlation diagram](/media/http-correlation.png)
+
+## How does this work?
+Three kind of correlation ID's are used to create the relationship:
+* **Transaction ID**: this ID is the one constant in the diagram. This ID is used to describe the entire transaction, from beginning to end. All telemetry will be tracked under this ID.
+* **Operation ID**: this ID describes a single operation within the transaction. This ID is used within a service to link all telemetry of that service together.
+* **Operation Parent ID**: this ID is create the parent/child link across services. When service A calls service B, then service A is the so called 'parent' of service B.
+
+The following list shows each step in the diagram:
+1. The initial call in this example doesn't contain any correlation headers. This can be seen as a first interaction call to a service. 
+2. Upon receiving at service A, the application will generate new correlation information. This correlation info will be used when telemetry is tracked on the service.
+3. When a call is made to service B, the **transaction ID** is sent but also the **operation parent ID** in the form of a hierarchical structure.
+4. The `jkl` part of this ID, describes the new parent ID for service B (when service B calls service C, then it will use `jkl` as parent ID)
+5. Service B responds to service A with the same information as the call to service B.
+6. The user receives both the **transaction ID** and **operation parent ID** in their final response.
+
+> Additional configuration is available to tweak this functionality, see the [dedicated Arcus Web API feature documentation](https://webapi.arcus-azure.net/features/correlation) for more in-depth information on HTTP correlation.
+
+## API Demonstration
+In this user-guide, two fictive API applications will be used to represent Service A and Service B of the diagram. Both services will be adapted to use service-to-service correlation.
+
+* **Product API** (Service A): receives an request to order a product if enough items of that product are in stock.
+* **Stock API** (Service B): response with the available items in stock of a given product
+
+The user interacts with the **Product API** to order their product. Internally, the **Product API** will contact the **Stock API** to verify if there are enough items of that product to complete the order. 
+
+> ⚠ Take into account that this sample should only be used for demonstration purposes and does not reflect a fully production-ready implementation. We have chosen to only provide the bare bones of the application so the changes are clear.
+
+> ⚠ Service-to-service correlation is currently only supported for projects that uses Application Insights as their Serilog logging system. If you want to use Arcus' service-to-service functionality outside Application Insights, [please let us know](https://github.com/arcus-azure/arcus.observability/issues/new/choose).
```

Comment:
This is already mentioned in the general instruction, I would remove it here as it does not directly have a link to the demo.

Reply:
Ah, yes, that's right. Let's remove that! 👍

Reply:
Done in https://github.com/arcus-azure/arcus.observability/pull/446/commits/ebf4f05e72e828bc7211458b22e9a59d5b64d123